src/services/hmwk-service.js

```diff
@@ -1,90 +1,96 @@
             mondayClient.setToken(token);
 
             const query = `query {
                 boards(ids: $studentsBoardId) {
-                  items {
-                    name
-              
-                    column_values {
-                      value
-                      text
+                    items {
+                        name
+            
+                        column_values {
+                            value
+                            text
+                        }
                     }
-                  }
                 }
-              }`
+            }`;
 
             const variables = { studentsBoardId };
 
             const response = await mondayClient.api(query, { variables });
-            const studentInfo = response.data.boards.items
-            return studentInfo //array of json
-            //student name: student_info[i].name 
+            const studentInfo = response.data.boards.items;
+            return studentInfo; //array of json
+            //student name: student_info[i].name
             //student email: student_info[i].column_values.text
-
         } catch (err) {
             console.log(err);
         }
     }
 
-    static async createNewHmwk(token, hmwkCompletionTrackingBoardId, hmwkName){
-        try{
+    static async createNewHmwk(token, hmwkCompletionTrackingBoardId, hmwkName) {
+        try {
             const mondayClient = initMondayClient();
             mondayClient.setToken(token);
-    
+
             //create a group at HmwkCompletionTrackingBoard with hmwkName
             const query = `mutation {
-                    create_group (board_id: $hmwkCompletionTrackingBoardId, group_name: $hmwkName) {
-                        id
-                    }
-                }`
-            
-            const variables = { hmwkCompletionTrackingBoardId,  hmwkName};
-    
+                create_group (board_id: $hmwkCompletionTrackingBoardId, group_name: $hmwkName) {
+                    id
+                }
+            }`;
+
+            const variables = { hmwkCompletionTrackingBoardId, hmwkName };
+
             await mondayClient.api(query, { variables });
-        }catch(err){
-            console.log(err)
+        } catch (err) {
+            console.log(err);
         }
     }
 
-    static async populateHmwkCompletionTrackingBoard(token, hmwkCompletionTrackingBoardId, studentInfo, hmwkName, dueDate){
-        try{
+    static async populateHmwkCompletionTrackingBoard(
+        token,
+        hmwkCompletionTrackingBoardId,
+        studentInfo,
+        hmwkName,
+        dueDate
+    ) {
+        try {
             const mondayClient = initMondayClient();
             mondayClient.setToken(token);
 
             //create a group at HmwkCompletionTrackingBoard with hmwkName
-            this.createNewHmwk(token, hmwkCompletionTrackingBoardId, hmwkName)
+            this.createNewHmwk(token, hmwkCompletionTrackingBoardId, hmwkName);
```

Comment:
`createNewHmwk()` is a static method. I think you're supposed to call it with

`HmwkService.createNewHmwk()`

and not

`this.createNewHmwk()`